config/prow/plugins.yaml

```diff
@@ -38,6 +38,11 @@ label:
     - component/event-notifier
     - component/pac-go-server
     - component/ui
+    - priority/backlog
```

Comment:
@Rajalakshmi-Girish can we use existing k8s labels? if yes - we may not need these entries here!